### ISSUE TITLE
No luarock module for cunn

### ISSUE THREAD

**Issue by prankkelkar** [llSourcell/how_to_convert_text_to_images#3]
(tensorflow) prasanna@prasanna-Inspiron-3558:~/tensorflow_programs/text$ sh demo/birds_demo.sh
/home/prasanna/torch/install/bin/lua: /home/prasanna/torch/install/share/lua/5.2/trepl/init.lua:389: module 'cunn' not found:No LuaRocks module found for cunn
	no field package.preload['cunn']
	no file '/home/prasanna/.luarocks/share/lua/5.2/cunn.lua'
	no file '/home/prasanna/.luarocks/share/lua/5.2/cunn/init.lua'
	no file '/home/prasanna/torch/install/share/lua/5.2/cunn.lua'
	no file '/home/prasanna/torch/install/share/lua/5.2/cunn/init.lua'
	no file '/home/prasanna/torch/install/lib/lua/5.2/cunn.lua'
	no file '/home/prasanna/torch/install/lib/lua/5.2/cunn/init.lua'
	no file './cunn.lua'
	no file '/home/prasanna/.luarocks/lib/lua/5.2/cunn.so'
	no file '/home/prasanna/torch/install/lib/lua/5.2/cunn.so'
	no file '/home/prasanna/torch/install/lib/cunn.so'
	no file '/home/prasanna/torch/install/lib/lua/5.2/loadall.so'
	no file './cunn.so'
stack traceback:
	[C]: in function 'error'
	/home/prasanna/torch/install/share/lua/5.2/trepl/init.lua:389: in function 'require'
	demo/get_embedding.lua:7: in main chunk
	[C]: in function 'dofile'
	...anna/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
	[C]: in ?
Traceback (most recent call last):
  File "demo/demo.py", line 15, in <module>
    from misc.config import cfg, cfg_from_file
ImportError: No module named 'misc'

**Comment by Saahiljeet-Singh** [comment]
@prankkelkar  : Did you find the solution to your problem ?
I am also facing the sam problem.
Also is CUDA a mandate to run this as I my system doesn't have an Nvidia GPU card.

**Comment by prankkelkar** [comment]
@Saahiljeet-Singh yes CUDA currently works only on Nvidia gpu cards. I know it will be heart breaking to hear that. Therefore now all new MacBooks are useless for training

**Comment by RajaUsman009** [comment]
How To download dataset?
